ipatests/test_integration/test_installation.py

```diff
@@ -2123,3 +2123,25 @@ def test_hostname_matching_domain(self):
         assert result.returncode == 1
         assert 'hostname cannot be the same as the domain name' \
             in result.stderr_text
+
+
+class TestPACDefaultConfig(IntegrationTest):
+    """Test PAC Default Configuration"""
+
```

Comment:
Instead of adding a new test class, you can add the test in TestInstallMaster. This spares a full server installation that takes around 10-15minutes.